src/measure_theory/bochner_integration.lean

```diff
@@ -2,11 +2,63 @@
 Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Zhouhang Zhou
-
-Bochner integral on real normed space
 -/
 
 import measure_theory.simple_func_dense
+import analysis.normed_space.bounded_linear_maps
+
+/-!
+# Bochner integral
+
+Bochner integral extends the definition of Lebesgue integral to functions that map froma  measure
+space into a Banach space (complete normed vector space). It is constructed here by extending the
+integral on simple functions.
+
+## Main definitions
+
+Bochner integral is defined following these steps:
+
+Step 1: Define integral on simple functions of the type `simple_func α β` (notation : `α →ₛ β`)
+  where `β` is a real normed space.
+
+  (See `simple_func.bintegral` and section `bintegral` for details. Also see `simple_func.integral`
+  for integral on simple functions of the type `simple_func α ennreal`.)
+
+Step 2: Use `simple_func α β` to cut out the simple functions from L1 functions, and define integral
+  on these. Simple functions in L1 space is written as `α →₁ₛ β`.
+
+Step 3: Show that the embedding of `α →₁ₛ β` into L1 is a dense and uniform one.
+
+Step 4: Show that the integral defined on `α →₁ₛ β` is a continuous linear map.
+
+Step 5: Define the bochner integral on L1 functions by extending the integral on integrable simple
```

Comment:
```suggestion
Step 5: Define the Bochner integral on L1 functions by extending the integral on integrable simple
```

Reply:
"Bochner" should be capitalized in all docstrings (when not referencing a name in Lean code).